src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newArr = [];
```

Comment:
1. [Checklist]  Make names descriptive, so they explain the purpose of variables and functions.
You can use some more  informative name for "newArr" variable (for example: newStates or newStatesArr).